src/constants/sort.ts

```diff
@@ -0,0 +1,5 @@
+export const SORT_TYPES = {
+  BEST_MATCH: "Best Match",
+  MOST_STARS: "Most Stars",
+  FEWEST_STARS: "Fewest Stars",
```

Comment:
Never define labels in enums. It should be:
```js
BEST_MATCH: "BEST_MATCH",
```

Infact you should define enums as:
```ts
enum SOME_ENUM {
   ENUM1 = "ENUM1",
   ENUM2 = "ENUM2"
}
```

Reply:
In your component you can create a constant for enum vs label like:
```js
const SOME_ENUM_VS_LABEL = {
  [SOME_ENUM.ENUM1]: "Some label 1"
}
```

Reply:
Fix this